lib/ui/settings/SettingsHomePanel.dart

```diff
@@ -941,16 +1015,25 @@ class _SettingsHomePanelState extends State<SettingsHomePanel> implements Notifi
   // Account
 
   Widget _buildAccount() {
-    return _OptionsSection(
-      title: Localization().getStringEx("panel.settings.home.account.title", "Your Account"),
-      widgets: <Widget>[
-        RibbonButton(
+    List<Widget> contentList = new List();
+
+    List<dynamic> codes = FlexUI()['settings.account'] ?? [];
+    for (int index = 0; index < codes.length; index++) {
+      String code = codes[index];
+      BorderRadius borderRadius = _borderRadiusFromIndex(index, codes.length);
+      if (code == 'personal_info') {
+        contentList.add(RibbonButton(
           height: null,
           border: Border.all(color: Styles().colors.surfaceAccent, width: 0),
-          borderRadius: _borderRadiusFromIndex(0, 1),
+          borderRadius: borderRadius,
```

Comment:
@TodorBachvarov Please review this change is correct. It's related to a previous fix made by you. Awaiting for confirmation